app/models/user.rb

```diff
@@ -310,19 +310,8 @@ def questions
     Node.questions.where(status: 1, uid: id)
   end
 
-  def content_followed_in_period(start_time, end_time)
```

Comment:
hi @rachaelkiteme , travis is showing 2 errors because you are removing this method, actually this method is used for generation of our digest page, so can't be removed